tests/ottest/fixtures/blockchain/RegtestSimple.cpp

```diff
@@ -446,6 +449,7 @@ void Regtest_fixture_simple::MineTransaction(
     auto send_transaction =
         user.api_->Crypto().Blockchain().LoadTransactionBitcoin(
             transactions_to_confirm);
+    transactions_.emplace_back(send_transaction->ID());
     transactions.emplace_back(std::move(send_transaction));
```

Comment:
why this member and local vector have same name which is suggesting the same type but it is not? can we do some kind of renaming which will suggest a little what this vecvtor is storing?